Lower minimum TTL to 600s

fabiante opened the pull request:
This fixes that records can't be set with a TTL of 3600 to 10800.

Practice has shown that the Hosttech API does in fact support a TTL of 3600 which is why this client library should support that.

MasterEvarior commented:
Hi @fabiante 

Thanks for your PR.
> Practice has shown that the Hosttech API does in fact support a TTL of 3600 which is why this client library should support that.  

Yep, you're right. It seems like they have changed something a while ago. I can't find it anywhere now but for a certain time there was a limit at 10800, which now seems to be gone.

The PR looks fine by me and the tests are OK. I'd appreciate it if you could also update the README to be current.

fabiante commented:
@MasterEvarior Sure. The readme states that at least 600 seconds must be used. According to the Hosttech DNS GUI I have always worked with a TTL of 3600, because the UI does not give smaller options.

I have tested the API and it seems the value from the README is the proper one:

![image](https://github.com/user-attachments/assets/2ac2c773-3fd0-44a6-a3b0-a34634cc5e2b)

Thus, I will now change the code to use 600 and update the README to reflect the actual behavior.

fabiante committed:
Lower minimum TTL to 600s

This fixes that records can't be set with a TTL between 600 to 10800.

Practice has shown that the Hosttech API does in fact support a TTL of 3600 which is why this client library should support that.

fabiante committed:
Update TTL documentation in README.md
